run.sh

```diff
@@ -1,14 +1,59 @@
-source venv/bin/activate
-
-# Run Spark Job
-PYSPARK_PYTHON=./VENV/venv/bin/python3 ../spark/spark-2.4.0-bin-hadoop2.7/bin/spark-submit \
---master yarn \
---conf spark.pyspark.virtualenv.enabled=true \
---conf spark.pyspark.virtualenv.type=native \
---conf spark.pyspark.virtualenv.requirements=requirements.txt \
---conf spark.pyspark.virtualenv.bin.path=./VENV/venv/bin/virtualenv \
---conf spark.yarn.appMasterEnv.PYSPARK_PYTHON=./VENV/venv/bin/python \
---archives venv.zip#VENV \
-src/spark.py --es "$1"
-
-deactivate
+#!/bin/bash
+
+#default values
+ES_PATH=`cat .es_path`
+INPUT_PATH="data/sample.warc.gz"
+OUTPUT_FILE="output.tsv"
+
+# check for input parameters
+while [[ $# -gt 0 ]]
+do
+case $1 in
+    -es)
+    ES_PATH="$2"
+    shift
+    shift
+    ;;
+    -f)
+    INPUT_PATH="$2"
+    shift
+    shift
+    ;;
+    -o)
+    OUTPUT_FILE="$2"
```

Comment:
Is this OUTPUTFILE == INPUTFILE?

Reply:
In -o) you can specify where you want the output to be.
Eg:

sh run.sh -o out111.tsv

This will make sure the output goes to out111.tsv instead of default "output.tsv"